README.md

```diff
@@ -926,12 +939,33 @@ INFO     Modules available from nf-core/modules (master)
 
 ### List installed modules
 
-To list modules installed in a local pipeline directory you can use `nf-core modules list --installed <directory>`.
+To list modules installed in a local pipeline directory you can use `nf-core modules list local`. This will list the modules install in the current working directory by default. If you want to specify another directory, use the `--dir <pipeline_dir>` flag.
 
-### Install a module into a pipeline
+```console
+$ nf-core modules list local
+                                          ,--./,-.
+          ___     __   __   __   ___     /,-._.--~\
+    |\ | |__  __ /  ` /  \ |__) |__         }  {
+    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
+                                          `._,._,'
+
+    nf-core/tools version 2.0
+
+
+INFO     Modules installed in '.':
+
+┏━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ Module Name ┃ Repository      ┃ Version SHA                              ┃ Message                                                                       ┃ Date       ┃
+┡━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━┩
+│ fastqc      │ nf-core/modules │ e937c7950af70930d1f34bb961403d9d2aa81c7d │ Rename software/ directory to modules/ to re-organise module structure (#567) │ 2021-07-07 │
+│ multiqc     │ nf-core/modules │ e937c7950af70930d1f34bb961403d9d2aa81c7d │ Rename software/ directory to modules/ to re-organise module structure (#567) │ 2021-07-07 │
+└─────────────┴─────────────────┴──────────────────────────────────────────┴───────────────────────────────────────────────────────────────────────────────┴────────────┘
```

Comment:
I know it's annoying but can you try to format this a little bit nicer? You can just delete some of the text and replace by `...` and then cut back some of the boxes. Just so that it doesn't wrap. Would have done it here but is difficult with the small suggestion box.

Reply:
Yes of course!